app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { data: { turbo: false} }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+    <div class="card container" style="padding: 3rem; max-width: 700px; margin: 1rem auto;">
+      <h2 class="text-center" style="margin: 0 0 1rem 0;">Sign up</h2>
+
+      <div class="mb-3">
+        <%= f.label :name, "Name", class: "form-label fs-6" %><br>
+        <%= f.text_field :name, :placeholder => "Add Full name...", required: true, class: "form-control" %><br>
+      </div>
+
+      <div class="mb-3">
+        <%= f.label :email, class: "form-label fs-6" %><br />
+        <%= f.email_field :email, autofocus: true, autocomplete: "email", class: 'form-control' %>
+      </div>
+
+      <div class="mb-3">
+        <%= f.label :password, class: "form-label fs-6" %>
+        <% if @minimum_password_length %>
+        <em>(<%= @minimum_password_length %> characters minimum)</em>
+        <% end %><br />
+        <%= f.password_field :password, autocomplete: "new-password", class: 'form-control' %>
+      </div>
+
+      <div class="mb-3">
+        <%= f.label :password_confirmation, class: "form-label fs-6" %><br />
+        <%= f.password_field :password_confirmation, autocomplete: "new-password", class: 'form-control' %>
+      </div>
```

Comment:
- [ ] **[OPTIONAL]** While these inputs are rightly validated at the model level, it would be nice to also add HTML validation for them by adding the `required: true` attribute for a much richer user experience.

Reply:
Implemented